tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/contact/EmailAddressContactSearchEngineContract.scala

```diff
@@ -178,4 +178,58 @@ trait EmailAddressContactSearchEngineContract {
     assertThat(SFlux.fromPublisher(testee().autoComplete(accountId, "rie")).asJava().map(_.fields).collectList().block())
       .containsOnly(contactFieldsB, otherContactFields)
   }
+
+  @Test
+  def deleteShouldDeletePersonalContact(): Unit = {
+    SMono(testee().index(accountId, otherContactFields)).block()
+
+    SMono(testee().delete(accountId, otherMailAddress)).block()
+
+    assertThat(SFlux.fromPublisher(testee().autoComplete(accountId, "pen")).asJava().map(_.fields).collectList().block())
```

Comment:
```suggestion
    assertThat(SFlux.fromPublisher(testee().autoComplete(accountId, "Car")).asJava().map(_.fields).collectList().block())
```
A reminder here: for surname and firstname we use edge_ngram